tests/test_tasks.py

```diff
@@ -146,6 +146,40 @@ def test_tasks(app, db, halt_workflow, sample_records_uri):
         assert job.status == JobStatus.ERROR
 
 
+def test_submit_result_starts_a_workflow(
+    app,
+    db,
+    halt_workflow,
+    sample_records_uri,
+    sample_records
+):
+    """Test tasks."""
+    job_id = uuid.uuid4().hex  # init random value
+    with app.app_context():
+        CrawlerJob.create(
+            job_id=job_id,
+            spider="Test",
+            workflow=halt_workflow.__name__,
+            logs=None,
+            results=None,
+        )
+        db.session.commit()
```

Comment:
This commit is not required, pls remove it.
Also, changing the state of the db in a test is bad practice as it affects other tests.

Reply:
PS: if you feel proactive you can try to remove the same statement in all other tests and see it was required for the test to pass. My guess is that in most cases, it is not.

Reply:
I don't believe that changing the DB state is a bad practice. Not cleaning it, instead, it is.
A proper `tear-down` function is defined in the `config.py` and run after each test, this ensures the DB consistency for each test. Also, not committing the session won't insert a CrawlerJob in the DB, making these tests fail because they read it from the DB.